webpack/components/AnsibleRolesSwitcher/components/AvailableRolesList.js

```diff
@@ -19,7 +19,9 @@ const AvailableRolesList = ({
       <Pagination
         viewType="list"
         itemCount={itemCount}
-        pagination={pagination}
+        updateParamsByUrl={false}
+        page={pagination.page}
+        perPage={pagination.perPage}
```

Comment:
if you'll just won't pass it, it will use the default per_page from:

https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/Pagination/index.js#LL34C42-L34C54

Reply:
@Ron-Lavi I'm getting more confused about how this should work... I couldn't find a better solution, where I don't pass these three properties to the Pagination component.

Reply:
P.S. I used the implementation [here](https://github.com/theforeman/foreman/blob/7e4d9b29f193cec9fa711e52eb7fdfc9d778937b/webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Parameters/ParametersTable.js#LL100C14-L100C24) as an example @MariaAga suggested.